src/file-system/index.js

```diff
@@ -0,0 +1,4 @@
+'use babel'
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Again, personally I'd split this up so you don't have to disable the rule and future work is simpler.

Reply:
And again this file will be exclusively (or close to it) exporting other files to create a single entry point to an api of related code, so the future work is just deleting the `eslint-disable`.